types/indefinite/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for indefinite 2.2
+// Project: https://github.com/tandrewnichols/indefinite
+// Definitions by: My Self <https://github.com/me>
```

Comment:
Please replace the placeholder values with your name / username. Thanks!

Reply:
Replaced